### ISSUE TITLE
Clarification of abbreviations used in Property Units

### ISSUE THREAD

**Issue by guptap1** [usnistgov/REFPROP-issues#61]

**Comment by guptap1** [comment]
The following may be helpful in clarification of units for certain properties:

Viscosity				uPa.s		means	 micro Pascal.Second, the numeric value should be divided by 1,000,000 to get Pa.Second
Specific Heat			(kJ/kg)/K		means 	kilo Joule per kg per degree Kelvin, the numeric value should be multiplied by 1,000 to get Joule per kg per degree K
Thermal Conductivity	mW/m-K		means 	milliwatts per meter per degree Kelvin, the numeric value should be divided by 1,000 to get Watts per meter per degree K

In other words, u in viscosity means micro, kJ in specific heat means kilo Joule, and mW in thermal conductivity means milliwatt.

**Comment by EricLemmon** [comment]
That is completely correct.  In the manual or the Refprop.for file in your Refprop directory you can see a list of all the units available.   I just updated the Refprop.for with a few comments on the actual units, as shown below.
```
!    ========== ============================================== =======================  ========================
!    Label      Description                                    SI Molar Units           SI Mass Units
!    ---------- ---------------------------------------------- -----------------------  ------------------------
!
!     Regular properties
!    -----------------------------------------------------------------------------------------------------------
!    T          Temperature                                    [K]                      [K]
!    P          Pressure (kilopascals)                         [kPa]                    [kPa]
!    D          Density (moles per cubic decimeter)            [mol/dm^3]               [kg/m^3]
!    V          Volume                                         [dm^3/mol]               [m^3/kg]
!    E          Internal energy                                [J/mol]                  [kJ/kg]
!    H          Enthalpy                                       [J/mol]                  [kJ/kg]
!    S          Entropy (joules per mole-kelvin)               [J/(mol*K)]              [(kJ/kg)/K]
!    CV         Isochoric heat capacity                        [J/(mol*K)]              [(kJ/kg)/K]
!    CP         Isobaric heat capacity                         [J/(mol*K)]              [(kJ/kg)/K]
!    CP/CV      Heat capacity ratio                            [-]                      [-]
!    W          Speed of sound (meters per second)             [m/s]                    [m/s]
!    Z          Compressibility factor                         [-]                      [-]
!    JT         Isenthalpic Joule-Thomson coefficient          [K/kPa]                  [K/kPa]
!    A          Helmholtz energy                               [J/mol]                  [kJ/kg]
!    (which is equal to A0+AR below, or R*T*(PHIG00+PHIR00)
!    G          Gibbs energy                                   [J/mol]                  [kJ/kg]
!    R          Gas constant                                   [J/(mol*K)]              [(kJ/kg)/K]
!    M          Molar mass (or of the mixture)                 [g/mol]                  [g/mol]
!    QMASS      Quality (not implemented, q not known)         N.A.                     [kg/kg]
!    QMOLE      Quality (not implemented, q not known)         [mol/mol]                N.A.

!    Transport, etc.
!    -----------------------------------------------------------------------------------------------------------
!    VIS        Viscosity (micropascals per second)            [uPa*s]                  [uPa*s]
!    TCX        Thermal conductivity (watts per meter-kelvin)  [W/(m*K)]                [W/(m*K)]
!    PRANDTL    Prandlt number                                 [-]                      [-]
!    TD         Thermal diffusivity                            [cm^2/s]                 [cm^2/s]
!    KV         Kinematic Viscosity                            [cm^2/s]                 [cm^2/s]
!    STN        Surface tension  (millinewtons per meter)      [mN/m]                   [mN/m]
!    DE         Dielectric constant                            [-]                      [-]

**Comment by guptap1** [comment]
Eric,

I thought the pressure is in Mega Pascal, MPa. This is what makes sense to me when computing the properties in Excel worksheet. For example, the saturation temperature for liquid oxygen at 0.101 MPa (which is atmospheric pressure) is 90 Kelvin. Please confirm.

Thanks.

Pradeep
Wed 24Oct18 15:29

**Comment by guptap1** [comment]
Eric,

I am still confused by units of pressure. When I run REFPROP in interactive mode, the pressure appears in Mega Pascal. Can you please explain what do you mean by “default units”. When running REFPROP in interactive model, are the units different from the defaults you have stated?

Thanks for you help.

Pradeep
Sat 03Nov18 21:46

**Comment by EricLemmon** [comment]
Hi Pradeep,

Within the internal code in Refprop, pressure is always used in kPa, but outside of that it could be anything depending on the user's wishes.  This of course means that caution must be used with all calculations, especially with energy, entropy, and entropy where it is not easy to have a "feeling" of what the magnitude should be since the reference state can also change.  

I'm not sure if the following helps with your questions, but below are the unit systems that you can select when calling either the ALLPROPS or REFPROP subroutines.  The first one, labeled DEFAULT with iUnits=0, should match those we've been discussing above from the ALLPROPS routine.  All of the others are systems that you can select at anytime to change the units to those you would like to use.

Eric

```
!    The unit systems used in Refprop are as follows::
!
!
!                         DEFAULT     MOLAR SI    MASS SI       SI WITH C
!        iUnits --->      0           1           2             3
!        Temperature      K           K           K             C
!        Pressure         kPa         MPa         MPa           MPa
!        Density          mol/dm^3    mol/dm^3    kg/m^3        kg/m^3
!        Enthalpy         J/mol       J/mol       J/g           J/g
!        Entropy          (J/mol)/K   (J/mol)/K   (J/g)/K       (J/g)/K
!        Speed            m/s         m/s         m/s           m/s
!        Kinematic vis.   cm^2/s      cm^2/s      cm^2/s        cm^2/s
!        Viscosity        uPa-s       uPa-s       uPa-s         uPa-s
!        Thermal cond.    W/(m-K)     mW/(m-K)    mW/(m-K)      mW/(m-K)
!        Surface tension  N/m         mN/m        mN/m          mN/m
!        Molar Mass       g/mol       g/mol       g/mol         g/mol
!
!                         MOLAR       MASS
!                         BASE SI     BASE SI     ENGLISH       MOLAR ENGLISH
!        iUnits --->      100         101         5             6
!        Temperature      K           K           F             F
!        Pressure         Pa          Pa          psia          psia
!        Density          mol/m^3     kg/m^3      lbm/ft^3      lbmol/ft^3
!        Enthalpy         J/mol       J/kg        Btu/lbm       Btu/lbmol
!        Entropy          (J/mol)/K   (J/kg)/K    (Btu/lbm)/R   (Btu/lbmol)/R
!        Speed            m/s         m/s         ft/s          ft/s
!        Kinematic vis.   m^2/s       m^2/s       ft^2/s        ft^2/s
!        Viscosity        Pa-s        Pa-s        lbm/(ft-s)    lbm/(ft-s)
!        Thermal cond.    W/(m-K)     W/(m-K)     Btu/(h-ft-R)  Btu/(h-ft-R)
!        Surface tension  N/m         N/m         lbf/ft        lbf/ft
!        Molar Mass       kg/mol      kg/mol      lbm/lbmol     lbm/lbmol
!
!                         MKS         CGS         MIXED         MEUNITS
!        iUnits --->      7           8           9             10
!        Temperature      K           K           K             C
!        Pressure         kPa         MPa         psia          bar
!        Density          kg/m^3      g/cm^3      g/cm^3        g/cm^3
!        Enthalpy         J/g         J/g         J/g           J/g
!        Entropy          (J/g)/K     (J/g)/K     (J/g)/K       (J/g)/K
!        Speed            m/s         cm/s        m/s           cm/s
!        Kinematic vis.   cm^2/s      cm^2/s      cm^2/s        cm^2/s
!        Viscosity        uPa-s       uPa-s       uPa-s         cpoise
!        Thermal cond.    W/(m-K)     mW/(m-K)    mW/(m-K)      mW/(m-K)
!        Surface tension  mN/m        dyne/cm     mN/m          mN/m
!        Molar Mass       g/mol       g/mol       g/mol         g/mol
!
!                         USER (can be changed by calling the REFPROP subroutine)
!        iUnits --->      11
!        Temperature      C
!        Pressure         psig
!        Density          kg/m^3
!        Enthalpy         J/g
!        Entropy          (J/g)/K
!        Speed            m/s
!        Kinematic vis.   cm^2/s
!        Viscosity        mPa-s
!        Thermal cond.    W/(m-K)
!        Surface tension  N/m
!        Molar Mass       g/mol
